CMakeLists.txt

```diff
@@ -397,6 +397,7 @@ find_package (ragel 6.10 REQUIRED)
 find_package (Threads REQUIRED)
 find_package (PthreadSetName REQUIRED)
 find_package (Valgrind REQUIRED)
+find_package (libbacktrace REQUIRED)
```

Comment:
as it depends on a third party library which is normally not packaged by the distro. IMHO, this feature / dependency should be optional. it should follow the model of how dpdk is supported.